imaginate_api/demo.py

```diff
@@ -1,5 +1,5 @@
-from ImageHandler import ImageHandler
-from schemas.image_info import ImageInfo
+from imaginate_api.image_handler import ImageHandler
+from imaginate_api.schemas.image_info import ImageInfo
```

Comment:
```suggestion
from image_handler_client.schemas.image_info import ImageInfo
```

Since the schema has been moved to that repository